packages/react-ui-core/src/Photo/Photo.js

```diff
@@ -109,14 +109,22 @@ export default class Photo extends PureComponent {
   fallback(e) {
     const { fallbackUrl } = this.props
 
-    if (e.target.src !== fallbackUrl) {
-      e.target.src = fallbackUrl
-      // force a re-render
+    if (e.target.src !== fallbackUrl && !this.state.error) {
+      if (fallbackUrl) {
```

Comment:
That would have probably been better 🤦‍♂️ . I'll try to add it in with the current PR I have out. The use case is that we _are_ adding an error class to the image to indicate something has failed.  I guess we don't have to do that. It was more so they could replace the image with some css instead of a fallbackUrl if they wanted.  Thoughts?